clarity.d.ts

```diff
@@ -251,7 +254,7 @@ interface IViewportState {
 
 declare const enum Instrumentation {
   JsError,
-  ApiMissing,
+  MissingFeature,
```

Comment:
At the moment we front-load check for all required features and agreed that's not ideal. In subsequent iterations each plug-in should be aware of it's own configuration (instead of hard coded values or using library-wide configurations) and be responsible for it's own check and instrumentation. However, for this PR, I'll prefer to scope to refactoring and add improve code in subsequent PRs.